ui/client/analytics/MatomoAnalytics.js

```diff
@@ -0,0 +1,20 @@
+import SystemUtils from "../common/SystemUtils";
+import * as MatomoTracker from "matomo-tracker";
+import AnalyticsEngine from "./AnalyticsEngine";
+
+export default class MatomoAnalytics extends AnalyticsEngine {
+
+  constructor(analyticsSettings) {
+    super();
+    this.siteId = analyticsSettings.siteId
+    this.url = analyticsSettings.url
```

Comment:
You cane use _.assign: https://lodash.com/docs/4.17.15#assign.

Reply:
It's simple assignment to class fields. Why lodash is better than simple = ?

Reply:
One line :) _.assign(this, _.pick(analyticsSettings, ["siteId", "url"]))

Reply:
Ok, done